### DISCUSSION TITLE
Deployment Error - SpecialFeatureOrQuotaIdRequired

### DISCUSSION THREAD

**Discussion by chadamplifiedit** [microsoft/PubSec-Info-Assistant#606] [General]
I'm attempting to start over with the latest version of this project, but I keep getting the following error when I attempt to make deploy.
I am able to manually create OpenAI resources.

SpecialFeatureOrQuotaIdRequired - The subscription does not have QuotaId/Feature required by SKU 'S0' from kind 'OpenAI' or contains blocked QuotaId/Feature.

**Comment by lon-tierney** [comment]
@chadamplifiedit please let us know if this issue is resolved for you via Support.
